ui/src/components/chart-view/QasChartView.vue

```diff
@@ -308,6 +320,16 @@ export default {
           label: this.title
         }
       }
+    },
+
+    parentComponent () {
+      return {
+        is: this.useBox ? 'qas-box' : 'div',
```

Comment:
Se tem o is no elemento, por que colocou aqui também?

Reply:
ele colocou aqui justamente para definir qual o `is` do component que ele vai usar